tree/dataframe/test/dataframe_nodes.cxx

```diff
@@ -47,15 +48,17 @@ TEST(RDataFrameNodes, RLoopManagerGetLoopManagerUnchecked)
    ASSERT_EQ(&lm, lm.GetLoopManagerUnchecked());
 }
 
-TEST(RDataFrameNodes, RLoopManagerJit)
+TEST(RDataFrameNodes, RLoopManagerJitWrongCode)
 {
    ROOT::Detail::RDF::RLoopManager lm(nullptr, {});
    lm.ToJitExec("souble d = 3.14");
-   auto op = [&](){
-      testing::internal::CaptureStderr();
+   bool caught = false;
+   try {
       lm.Run();
-   };
-   EXPECT_ANY_THROW(op()) << "Bogus C++ code was jitted and nothing was detected!";
+   } catch (const std::runtime_error &e) {
+      caught = true;
+   }
+   EXPECT_TRUE(caught) << "Bogus C++ code was jitted and nothing was detected!";
```

Comment:
Why not
```
EXPECT_THROW(lm.ToJitExec("souble d = 3.14"), std::runtime_error) << "Bogus C++ code was jitted, expect this to throw!";
```
Also here, it will obviously also work in the current implementation.

Reply:
because I didn't know `EXPECT_THROW` :D thanks